taxcalc/policy_current_law.json

```diff
@@ -56,16 +56,16 @@
         "section_2": "Social Security FICA",
         "irs_ref": "W-2, Box 4, instructions",
         "notes": "This parameter is indexed by the rate of growth in average wages, not by the price inflation rate.",
-        "row_label": ["2013",
-                      "2014",
-                      "2015",
-                      "2016",
-                      "2017",
-                      "2018"],
+        "value_yrs": [2013,
+                      2014,
+                      2015,
+                      2016,
+                      2017,
+                      2018],
         "indexable": true,
         "indexed": true,
-        "col_var": "",
-        "col_label": [],
+        "vi_name": "",
+        "vi_vals": [],
```

Comment:
@martinholmer, what does "vi" stand for?

Reply:
`vi` stands for `vector index`.

Reply:
Ah, I see. Thanks.